Examples/MagicWeather/MagicWeather/Sources/Controllers/PaywallViewController.swift

```diff
@@ -59,15 +59,15 @@ class PaywallViewController: UITableViewController {
         
         /// - Configure the PackageCell to display the appropriate name, pricing, and terms
         if let package = self.offering?.availablePackages[indexPath.row] {
-            cell.packageTitleLabel.text = package.product.localizedTitle
+            cell.packageTitleLabel.text = package.storeProduct.localizedTitle
             cell.packagePriceLabel.text = package.localizedPriceString
             
-            if let intro = package.product.introductoryPrice {
-                if intro.price == 0 {
-                    cell.packageTermsLabel.text = "\(intro.subscriptionPeriod.periodTitle()) free trial"
-                } else {
-                    cell.packageTermsLabel.text = "\(package.localizedIntroductoryPriceString) for \(intro.subscriptionPeriod.periodTitle())"
-                }
+            if let intro = package.storeProduct.introductoryDiscount {
+                let packageTermsLabelText = intro.price == 0
+                ? "\(intro.subscriptionPeriod.periodTitle()) free trial"
+                : "\(package.localizedIntroductoryPriceString ?? "unknown") for \(intro.subscriptionPeriod.periodTitle())"
```

Comment:
this is the default indentation, but I don't love it, happy to update

Reply:
Ugh that's an Xcode regression 🥺